perfectscale/manifest.json

```diff
@@ -49,6 +49,23 @@
   "assets": {
     "dashboards": {
       "PerfectScale": "assets/dashboards/perfectscale_alerts.json"
+    },
+    "integration": {
+      "auto_install": true,
+      "source_type_id": "",
```

Comment:
This is a new required field. To resolve the errors, you can run `ddev create` again and the field should be populated. Copy the source_type_id that is generated and paste into the field here! Let me know if you run into issues.

Reply:
Hi @emarsha94 
I updated by ddev to 9.1.0 and ran ddev create command and there was no such ID in the manifest that was created.

Reply:
Hi @idan-gazit Sorry for the delay here! I was testing on my end to find the solution. You will need to run the following: 
`ddev create -t check test`

Then copy the source type id and paste it in this file!

Reply:
Hi @emarsha94 Thanks for the tip !
Did just that, source id was added to the PR :)